cmd/limactl/start.go

```diff
@@ -29,16 +30,46 @@ import (
 
 func newStartCommand() *cobra.Command {
 	var startCommand = &cobra.Command{
-		Use:               "start NAME|FILE.yaml|URL",
-		Short:             fmt.Sprintf("Start an instance of Lima. If the instance does not exist, open an editor for creating new one, with name %q", DefaultInstanceName),
+		Use: "start NAME|FILE.yaml|URL",
+		Example: `
+Create an instance "default" (if not created yet) from the default Ubuntu template, and start it:
+$ limactl start
+
+Create an instance "default" from a template "docker":
+$ limactl start --name=default template://docker
```

Comment:
I wonder if the text should explain where the templates are located.

The README says:

> Older releases require `limactl start /usr/local/share/doc/lima/examples/TEMPLATE.yaml` instead.

which kind of documents it implicitly, but `limactl start --help` doesn't tell you which templates exist, not where they can be found.

Ideally (not in this PR) I would like to see something like `limactl start --list-templates` (similar to `limactl ls --list-fields`) that lists all the template names (maybe with a single-line summary for each, but maybe that doesn't make sense).

Reply:
Added `limactl start --list-templates`